app/components/spotlight/analytics/dashboard_component.html.erb

```diff
@@ -0,0 +1,51 @@
+<%= cache current_exhibit, expires_in: 1.hour do %>
+  <% if params[:start_date] || params[:end_date] %>
```

Comment:
I'm thinking we should remove the `params[:start_date]` and `params[:end_date]` functionality from this partial and from `dashboard_component.rb` until we can more completely think through the UI. The caching isn't set up right for this and we haven't thought through the controls or what to display in the UI -- displaying "User activity from 30daysAgo to today" doesn't seem great. I can create a new issue for this to discuss with Cathy and Alan. The rest of the code supports adding these features without too much trouble later.

Reply:
ok